brainstormforce/astra
# feat: PHP to ReactJS Select control.

### Description
Trello Task - https://trello.com/c/5v5QsFlC/16-ast-select

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
 - Check Backward Compatibility.
 - Check Options are saving are not.

### Checklist:
